lib/erb_lint/runner_config.rb

```diff
@@ -45,24 +45,29 @@ def merge!(other_config)
     end
 
     class << self
-      def default
+      def default(defaults_enabled: true)
         new(
           linters: {
-            AllowedScriptType: { enabled: true },
-            ClosingErbTagIndent: { enabled: true },
-            ExtraNewline: { enabled: true },
-            FinalNewline: { enabled: true },
-            NoJavascriptTagHelper: { enabled: true },
-            ParserErrors: { enabled: true },
-            RightTrim: { enabled: true },
-            SelfClosingTag: { enabled: true },
-            SpaceAroundErbTag: { enabled: true },
-            SpaceIndentation: { enabled: true },
-            SpaceInHtmlTag: { enabled: true },
-            TrailingWhitespace: { enabled: true },
+            AllowedScriptType: { enabled: defaults_enabled },
+            ClosingErbTagIndent: { enabled: defaults_enabled },
+            ExtraNewline: { enabled: defaults_enabled },
+            FinalNewline: { enabled: defaults_enabled },
+            NoJavascriptTagHelper: { enabled: defaults_enabled },
+            ParserErrors: { enabled: defaults_enabled },
+            RightTrim: { enabled: defaults_enabled },
+            SelfClosingTag: { enabled: defaults_enabled },
+            SpaceAroundErbTag: { enabled: defaults_enabled },
+            SpaceIndentation: { enabled: defaults_enabled },
+            SpaceInHtmlTag: { enabled: defaults_enabled },
+            TrailingWhitespace: { enabled: defaults_enabled },
           },
         )
       end
+
+      def apply_defaults(config)
+        default_linters_enabled = config["linters"]["DefaultLintersEnabled"]
```

Comment:
How about mirroring rubocop and naming this `DisabledByDefault` (normally under the `AllCops` config in a file)?

Reply:
We didn't do that because we didn't want users to think that there will be other options aside from `DisabledByDefault`. For example, people may use`{ AllCops: { Exclude: blablah } }` thinking it will work for AllCops, but it isn't implemented.

Reply:
That's fair. How about if it were under `linters`, would we worry about the same thing?

If we stick with this name, I'd tweak it a little. `DefaultLintersEnabled` sounds like it could be either a boolean or a list. Something like `EnableDefaultLinters` seems a little less ambiguous 🙂